x-pack/plugins/security_solution/public/common/components/matrix_histogram/index.tsx

```diff
@@ -315,25 +185,10 @@ export const MatrixHistogramComponent: React.FC<MatrixHistogramComponentProps> =
             toggleQuery={hideQueryToggle ? undefined : toggleQuery}
             subtitle={subtitleWithCounts}
             inspectMultiple
-            showInspectButton={showInspectButton && !isChartEmbeddablesEnabled}
+            showInspectButton={false}
```

Comment:
If we set `showInspectButton={false}` we should also remove `inspectMultiple` and `isInspectDisabled` props, for consistency.